### PR TITLE
Update linen_intro.ipynb

### PR DESCRIPTION
**Pull request by afrozenator**
typo and slight rewording of sentence -- it can still be written better I guess.

# What does this PR do?

typo and slight rewording of sentence -- it can still be written better I guess.

Fixes # (issue)

## Checklist
- [ x] This PR fixes a minor issue (e.g.: typo or small bug) or improves the docs (you can dismiss the other
      checks if that's the case).
- [ ] This change is discussed in a Github issue/
      [discussion](https://github.com/google/flax/discussions) (please add a
      link).
- [ ] The documentation and docstrings adhere to the
      [documentation guidelines](https://github.com/google/flax/blob/master/docs/README.md#how-to-write-code-documentation).
- [ ] This change includes necessary high-coverage tests.
      (No quality testing = no merge!)

### CODE REVIEW

File: docs/notebooks/linen_intro.ipynb [hunk 1]
```diff
@@ -400,7 +400,7 @@
         "id": "uYwS5KbcmYIp"
       },
       "source": [
-        "Flax uses lazy initialization, which allows declared variables to be initialized only at the first site of their use, using whatever shape information is available a the local call site for shape inference.  One a variable has been initialized, and a reference to the data kept for use in subsequent calls.\n",
+        "Flax uses lazy initialization, which allows declared variables to be initialized only at the first site of their use, using whatever shape information is available a the local call site for shape inference.  Once a variable has been initialized a reference to the data is kept for use in subsequent calls.\n",
```
Comment (8bitmp3) [inline docs/notebooks/linen_intro.ipynb:403]: nit:

```suggestion
        "Flax uses lazy initialization, which allows declared variables to be initialized only at the first site of their use, using whatever shape information is available a the local call site for shape inference.  Once a variable has been initialized, a reference to the data is kept for use in subsequent calls.\n",
```

**Comment by andsteing** [comment]
Is this PR still alive?

**Comment by afrozenator** [comment]
@andsteing - I hope so? Someone has to approve this right?

**Review by jheek — APPROVED** [review]
Looks good, thanks!

**Pull request merged**